src/screens/internal/Anomalies/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { Annotations, Anomalies } from 'api/schema/RegionSummaryWithTimeseries';
+import { assert } from 'common/utils';
+
+// Not including all possible values because many fields never or
+// rarely get anomalies. We should extend this enum and related code as needed.
+export enum AnnotationType {
+  NEW_DEATHS,
+  NEW_CASES,
+  HOSPITALIZATIONS,
+}
+
+export const ANNOTATION_TYPES = [
+  AnnotationType.NEW_CASES,
+  AnnotationType.NEW_DEATHS,
+  AnnotationType.HOSPITALIZATIONS,
+];
+
+export const annotationTypeNames: { [metric in AnnotationType]: string } = {
+  [AnnotationType.NEW_CASES]: 'New Cases',
+  [AnnotationType.NEW_DEATHS]: 'New Deaths',
+  [AnnotationType.HOSPITALIZATIONS]: 'Hospitalizations',
+};
+
+export function getAnomaliesForAnnotationType(
+  annotations: Annotations,
+  type: AnnotationType,
+  orderAnomalies?: string,
+): Anomalies {
+  assert(ANNOTATION_TYPES.includes(type));
+  switch (type) {
+    case AnnotationType.NEW_CASES:
+      return annotations?.newCases?.anomalies ?? [];
+    case AnnotationType.NEW_DEATHS:
+      return annotations?.newDeaths?.anomalies ?? [];
+    case AnnotationType.HOSPITALIZATIONS:
+      return annotations?.hospitalBeds?.anomalies ?? [];
+  }
+}
+
+export interface AnnotationOptions {
+  snapshot: number;
+  stateFips: string;
+  annotationType: AnnotationType;
+}
+
+export interface AnnotationSelectorProps {
+  onNewOptions: (options: AnnotationOptions) => void;
+}
+
+export interface AnnotationSelectorInnerProps extends AnnotationSelectorProps {
+  mainSnapshot: number;
+}
```

Comment:
nit: I would move these two `Props` interfaces into AnomaliesSelector.tsx since they're used in exactly one place (defining the components) they're used and it's easier to understand the component if you can immediately see the props that it takes without going to a different file.

Moving to utils would make more sense to me if they were used in a bunch of places.